src/screens/search/blockFilters.tsx

```diff
@@ -0,0 +1,150 @@
+import { StyleSheet, View } from 'react-native'
+import React, { useState } from 'react'
+import { FlatList } from 'react-native-gesture-handler'
+import { Text, Divider, TouchableRipple, Switch, Button } from 'react-native-paper'
+import colors from '../../utils/colors'
+import { MaterialCommunityIcons } from '@expo/vector-icons'
+import { useNavigation } from '@react-navigation/core'
+import routes from '../../navigation/routes'
+import { useActionSheet } from '@expo/react-native-action-sheet'
+
+export const BlockFilters = () => {
+	const navigation = useNavigation()
+	const { showActionSheetWithOptions } = useActionSheet()
+
+	const [onlyStarred, setOnlyStarred] = useState(false)
+	const [sortIndex, setSortIndex] = useState(null)
+
+	const filters = [
+		{ key: 'sort', title: 'Sort' },
+		{ key: 'owner', title: 'Owner' },
+		{ key: 'blockType', title: 'Block Type' },
+		{ key: 'onlyStarred', title: 'Only Starred' },
+	]
+	const sortingOptions = ['Star count', 'Last updated', 'Creation date', 'Cancel']
```

Comment:
When "Cancel" is clicked, it's selected as the option (should just keep what's current)